### DISCUSSION TITLE
nan's in word_embedder() results only when using TK model

### DISCUSSION THREAD

**Discussion by pvonderlind** [sebastian-hofstaetter/teaching#110] [Advanced IR: Exercise Q&A] [answered]
Hi everyone, I currently have the issue that only when using the TK model,
the word_embedder most of the time just produces tensors conistsing of nan values.
This of course disables any learning, as the loss is always nan.

I did not change any of the initial template code regarding model creation, __init__ method etc.
Did anyone have similar issues?

**Comment by sophiaalthammer** [comment, marked as answer]
If you didnt change anything else, the problem is probably in your forward function of the TK model! But without Looking at the code I can also not help you, I am sorry!

**Reply by pvonderlind** [reply]
The weird thing is, the problem occurs even before calling the forward of the model for the first time. I fixed this issue now by coding a workaround.
